Documentation/source/guide/tutorials-imagesequence.rst

```diff
@@ -75,24 +69,37 @@ Blender
 https://www.blender.org/
 
 - Import the movie file in Blender Video Sequencer.
-- Go to render properties
-- In Color management change view transform standard
-- GO to output properties
-- Select File format PNG/TIFF
+- Go to render properties.
+- In Color management change view transform standard.
+- Go to output properties.
+- Select File format PNG/TIFF.
 - Select RGB/RGBA, 8/16 Color depth, and preferred compression NONE/Any.
 
 Full instructions on how to use the Blender VSE can be found here https://docs.blender.org/manual/en/latest/video_editing/index.html
 
 
 Adobe Media Encoder
 ~~~~~~~~~~~~~~~~~~~
-- Open media encoder
-- Add source video to the queue
+- Open Media Encoder.
+- Add source video to the queue.
 - Set the output format to OpenEXR.
-- Set compression to "Zip"
-- If the source has an alpha channel be sure to scroll down to the bottom of the Video section of the Export Settings and check "Include Alpha Channel"
+- Set compression to "Zip".
+- If the source has an alpha channel be sure to scroll down to the bottom of the Video section of the Export Settings and check "Include Alpha Channel".
 - Close the Export Settings by clicking Ok and press the Start Queue button.
 
+
+DaVinci Resolve
+~~~~~~~~~~~~~~~~
+- In Resolve, select your videoclip.
+- Go to File => Media Management.
+- Select Clips.
+- Select Media Destination.
+- Select Video format => TIFF or EXR.
```

Comment:
There must be one step before this so the Video format setting can be available:
- Select "Transcode".
And that step should be placed before "Select Media Destination" in order of the interface.

The next step should be:
![image](https://user-images.githubusercontent.com/65440815/85194214-1d82d080-b2f7-11ea-928a-4e814dd80db0.png)
In case they may have used any media.

Should we include codec suggestion in our document?
![image](https://user-images.githubusercontent.com/65440815/85194322-9124dd80-b2f7-11ea-88fc-c2980e3c896d.png)